moto/rds/models.py

```diff
@@ -2669,9 +2669,15 @@ def restore_db_cluster_to_point_in_time(
         return self.create_db_cluster(new_cluster_props)
 
     def failover_db_cluster(
-        self, db_cluster_identifier: str, target_db_instance_identifier: str
+        self, db_cluster_identifier: str, target_db_instance_identifier: str = ""
     ) -> DBCluster:
         cluster = self.clusters[db_cluster_identifier]
+
+        if not target_db_instance_identifier:
+            for instance_id, instance in self.databases.items():
+                if not instance.is_cluster_writer:
+                    target_db_instance_identifier = instance_id
+
```

Comment:
Won't this blow up if called on a cluster that only has one member?  Can you find out what the AWS response is in that case and raise an appropriate exception here?